staging/src/k8s.io/code-generator/cmd/informer-gen/generators/generic.go

```diff
@@ -50,14 +51,11 @@ func (g *genericGenerator) Filter(c *generator.Context, t *types.Type) bool {
 }
 
 func (g *genericGenerator) Namers(c *generator.Context) namer.NameSystems {
-	pluralExceptions := map[string]string{
-		"Endpoints": "Endpoints",
```

Comment:
where have these gone?

Reply:
@sttts they are defaulted in flag

Reply:
actually, I forgot to copy some, good catch :-)

Reply:
fixed.

Reply:
Document it in the flag description.

Reply:
done